patches/CMakeLists.patch

```diff
@@ -0,0 +1,107 @@
+Index: jni/CMakeLists.txt
```

Comment:
Add directory "windows" after "patches". Also, can we maintain a document with a description for each patch?

Reply:
I refactored the file structure. We can actually add description at the top of each patch. I have added the description in CMakeLists.patch. Please check and suggest me if you want me to make any changes